Added labels

TobiA34 opened the pull request:
Add new salary labels underneath submit button.

TobiA34 committed:
Added new labels for the salaries

tunds commented on Salary App/ViewController.swift:
```diff
@@ -66,33 +70,33 @@ class ViewController: UIViewController, UITextFieldDelegate {
         case 2:
             let salary = Int(self.salaryTextField.text!) ?? 0
             weeklySalary =  (salary / 52)
             defaults.set(weeklySalary, forKey: "weekly")
```
Do we still need this?

tunds commented on Salary App/ViewController.swift:
```diff
@@ -13,8 +13,11 @@ class ViewController: UIViewController, UITextFieldDelegate {
     
     @IBOutlet weak var salarySC: UISegmentedControl!
     
+    @IBOutlet weak var monthlySalaryLbl: UILabel!
```
Rather than storyboards can we add all the UI programatically, use a stack view to hold all the views this will show you how to use stackviews for building UI components

tunds commented on Salary App/Base.lproj/Main.storyboard:
```diff
@@ -44,30 +44,59 @@
                                     <action selector="submitBtnPressed:" destination="BYZ-38-t0r" eventType="touchUpInside" id="TPH-sh-CCE"/>
                                 </connections>
                             </button>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="28" translatesAutoresizingMaskIntoConstraints="NO" id="MvT-zZ-4Lz">
```
Not many companies use storyboard find some resources on setting this up programatically

tunds commented on Salary App/ViewController.swift:
```diff
@@ -66,33 +70,33 @@ class ViewController: UIViewController, UITextFieldDelegate {
             let salary = Int(self.salaryTextField.text!) ?? 0
```
When working with money its better to use decimals, with int's it can only ever by a whole number what about if i get paid £500.50 using an Int will turn this into 500 which isn't correct

tunds commented on Salary App/ViewController.swift:
```diff
@@ -66,33 +70,33 @@ class ViewController: UIViewController, UITextFieldDelegate {
             let salary = Int(self.salaryTextField.text!) ?? 0
             monthlySalary =  salary / 12
```
Could we seperate out this calculation logic into some kinda of calculator object, pass the function an enum and based on that case do some calculations based on the salary you pass in.

Also it would allow you to write unit tests on it easily which is something you should look into next

TobiA34 committed:
Change from storyboard to be programmatic
Added code to format the currency
Added navigation title
Added new computed variables

TobiA34 committed:
Removed submit button
Added new colour set for dark mode
Add View model file

TobiA34 committed:
Added UI and Unit test
